docs/reference/gc-toolkit/onboarding/index.md

```diff
@@ -0,0 +1,130 @@
+- Comapeo: https://comapeo.alias.guardianconnector.net/ 
+- Windmill: https://windmill.alias.guardianconnector.net/ 
+
+## Authentication
+
+All the services are protected so no one can access them without your permission.
+
+The first barrier is your Alias. Please do not share this with anyone outside of your organization, as it is the first door to your infrastructure.
+
+We also have a user access list that we will experience in just a second.
+
+For some services, we might have a predefined user/password combination that I will share with you when we get to them.
+
+## Tools
+
+Guardian Connector is equipped with a suite of tools designed to help communities manage their data and workflows. An extensive walkthrough of these tools is available in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/ 
+
+### Explorer
+
+It is the service that hosts your Mining alerts, as well as your Comapeo projects. Right now you will only be able to see the mining alerts. Let’s take a look at that.
+
+Let’s go into https://explorer.alias.guardianconnector.net/ 
+
+#### First time login
+
+You will attempt to login. You can use a gmail account if you have one, or create an account for this service. If you do have a gmail account, it is usually easier to use that one as your login.
+
+Every time the system shows this screen, you will use the same authentication mechanism. It works across the services we provide.
+
+Your access attempt will fail, as you are not authorized yet to access the system. I will give you authorization to access now.
+
+Now you can attempt to access again.
+
+#### Overview
+
+Right now Explorer will only have one view, your current alerts.
+
+You can find more information about Explorer in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/gc-explorer/
+
+#### Whatsapp alerts
+
+#### Ways this space will grow
+
+As you grow in usage of the system, Explorer will host:
+
+- Your remote sensing alerts: logging, mining, airstrips and roads
+- Your views of the CoMapeo projects (both maps and media you share within it)
+
+#### Customization
+
+The way the map looks like in the alerts dashboard can be customizable.
+
+Let’s discuss what would be relevant for you to have there.
+
+### Files
+
+Files is where you can safely store your files. It is a less structured means for you to preserve your data, whatever it may be.
+Your current access credentials are:
+
+- URL: https://files.alias.guardianconnector.net/ 
+- User: 
+- Password: 
+
+You can find more information about it in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/filebrowser/
+
+#### Existing data
+
+You will notice that there are a few folders already available in your Files view, and that more might appear without you doing it.
+This happens because Files is also used for Guardian Connector tools to preserve data that you get from other services.
```

Comment:
```suggestion
You will notice that there are a few folders already available in your Filebrowser view, and that more might appear without you doing it.
This happens because Filebrowser is also used for Guardian Connector tools to preserve data that you get from other services.
```

Change everywhere